platform/scripts/os-check.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "[INFO]: Starting OS validation.."
+
+if [ -f /etc/os-release ];
+then
+  . /etc/os-release
+  OS=$NAME
+  VER=$VERSION_ID
+
+elif [ -f /etc/lsb-release ];
+then
+  . /etc/lsb-release
+  OS=$DISTRIB_ID
+  VER=$DISTRIB_RELEASE
+else
+  # Fall back to uname, e.g. "Linux <version>", also works for BSD, etc.
+  OS=$(uname -s)
+  VER=$(uname -r)
+fi
+
+if [[ ("${OS}" != "Ubuntu") && ("${OS}" != "Red Hat Enterprise Linux Server") ]];
```

Comment:
Lets make a list here so that modification is easier. We are currently testing centos 7.6 as well